generate.py

```diff
@@ -65,28 +66,47 @@ def generate_perf(conf):
     """
     shutil.copytree("perf/backup/", "perf/package/")
     generate_fernivy("perf",
-        lambda line :
-            line.replace(
-                    "# Request for sudo access when needed",
-                    "if (( $EUID != 0 )); then echo \"Please run as root!\"; exit; fi"
-                )
-                .replace("$TOOL\"/backup/\"$TOOL\"_run.sh\"", "/usr/lib/perf_run.sh")
-                .replace("parser.py", "/usr/lib/parser.py")
-    )
+                     lambda line:
+                     line.replace(
+                         "# Request for sudo access when needed",
+                         "if (( $EUID != 0 )); then echo \"Please run as root!\"; exit; fi"
+                     )
+                     .replace("$TOOL\"/backup/\"$TOOL\"_run.sh\"", "/usr/lib/perf_run.sh")
+                     .replace("parser.py", "/usr/lib/parser.py")
+                     )
     generate_perf_control(conf)
     shutil.copyfile("parser.py", "perf/package/parser.py")
 
 
-def generate_powerlog():
+def generate_powerlog(conf):
     """
-    This generator creates the package for powerlog.
+    This generator creates the package and package-brew for powerlog.
     """
     shutil.copytree("powerlog/backup/", "powerlog/package/")
+    shutil.copytree("powerlog/backup/", "powerlog/package-brew/")
+    # installation from source
+    generate_fernivy("powerlog",
+                     lambda line:
+                     line.replace(
+                         "$TOOL\"/backup/\"$TOOL\"_run.sh\"",
+                         "./powerlog/package/powerlog_run.sh")
+                     .replace("parser.py", "./powerlog/package/parser.py")
```

Comment:
hmm now that I think about it, I'm not sure if we wanna do this. when people install from source, do we want them to cd to `powerlog/package` and call it from there? or do we wanna have them call `powerlog/package/fernivy` from root?

Reply:
We actually want to be able to zip the `package` separately, so it should be relative to that.

Reply:
done:)